engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStoreBenchmark.scala

```diff
@@ -0,0 +1,52 @@
+package cromwell.engine.workflow.lifecycle.execution
```

Comment:
I think all that can be done yes. To be honest I did this more as a quick way to know if what I was doing was making any (good) difference and didn't plan on spending too much time on setting something CI automated up as part of this PR, but that would definitely be nice. Happy to make a ticket for it.
Currently you can run it directly through intellij or with sbt (I'll find the command). It's not run automatically.